meteor/lib/collections/ShowStyleVariants.ts

```diff
@@ -1,22 +1,33 @@
 import { Meteor } from 'meteor/meteor'
-import { ObserveChangesForHash, createMongoCollection } from './lib'
+import { createMongoCollection, ObserveChangesForHash } from './lib'
 import { registerIndex } from '../database'
-import { ShowStyleVariantId } from '@sofie-automation/corelib/dist/dataModel/Ids'
-export { ShowStyleVariantId }
+import { ShowStyleBaseId, ShowStyleVariantId } from '@sofie-automation/corelib/dist/dataModel/Ids'
 import { CollectionName } from '@sofie-automation/corelib/dist/dataModel/Collections'
 
 import { DBShowStyleVariant } from '@sofie-automation/corelib/dist/dataModel/ShowStyleVariant'
+
+export { ShowStyleVariantId }
 export * from '@sofie-automation/corelib/dist/dataModel/ShowStyleVariant'
 
 export { ShowStyleCompound } from '@sofie-automation/corelib/dist/dataModel/ShowStyleCompound'
 
 export type ShowStyleVariant = DBShowStyleVariant
 export const ShowStyleVariants = createMongoCollection<ShowStyleVariant>(CollectionName.ShowStyleVariants)
 
+export type ShowStyleVariantsOrder = { _id: ShowStyleVariantId; rank: number; showStyleBaseId: ShowStyleBaseId }
+
+export const OrderedShowStyleVariants = createMongoCollection<ShowStyleVariantsOrder>(
```

Comment:
Again with the broad name for a very specific collections. The point of making a separate collection disappears by making the name broad.